src/components/ButtonPanel.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+import './ButtonPanel.css';
+import './button.css';
+
+const ButtonPanel = ({ handleClick }) => (
+
+  <div className="panel">
+
+    <div className="group">
+      <Button handleClick={handleClick} color="light-gray" name="AC" />
```

Comment:
- [ ] [OPTIONAL] Think on a way to repeat less code, the <Button> element em specific, you can try adding a loop to make the code drier.